iib/web/api_v1.py

```diff
@@ -667,6 +674,7 @@ def regenerate_bundle():
         payload.get('organization'),
         request.id,
         payload.get('registry_auths'),
+        payload.get('bundle_replacements'),
```

Comment:
Could you please add documentation for this parameter. It should be under `iib/web/static/api_v1.yml`